lib/commander/version.rb

```diff
@@ -1,3 +1,3 @@
 module Commander
-  VERSION = '4.5.3'.freeze
+  VERSION = '4.6.0'.freeze
```

Comment:
I think independently bumping the Commander version number for our fork, as if this was the main Commander repo, is only likely to confuse us in future - maybe calling this `4.6.0-alces` would be clearer?

Reply:
I have updated to be `4.4.3-alces0`. Version `4.4.3` was the point in which we forked the project. It will be useful to maintain as a reference to where we are compared to the original project.